### DISCUSSION TITLE
Use existing modified gotk-components.yaml

### DISCUSSION THREAD

**Discussion by Forbzy** [fluxcd/flux2#1836] [Q&A] [answered]
Is there a way during the flux bootstrap for me to use an existing gotk-components.yaml file? Mine is always over written and pushed to Github. The reason I ask is I need to be able to modify the memory and CPU resource usage of the components to work in my setup but it always goes back to default.

**Comment by stefanprodan** [comment, marked as answer]
You can patch the components in a way that works with flux bootstrap, please see https://fluxcd.io/docs/installation/#customize-flux-manifests
